lega/ingest.py

```diff
@@ -57,10 +58,18 @@ def work(fs, inbox_fs, data):
     org_msg = data.copy()
     data['org_msg'] = org_msg
 
-    # Insert in database
+    # this uses a function to set the filepath and user_id
```

Comment:
Is this approach taken to defer changes to the database insert code? Otherwise, I'd prefer calculating the checksum (if needed) and making sure all goes in at the same time (extending db.insert_file to take the checksum as well.

Reply:
line 130 inserts the checksum ` db.set_file_encrypted_checksum(file_id, encrypted_checksum, encrypted_checksum_type)` 
I tried to touch as little code as possible from the original one, as I thought we agreed in planning
That meant not changing the db functions

Reply:
I would prefer to insert it at the same time, but did not want to touch db, yet. 
I am afraid if we start modifying db we will go down a rabbit hole and end up re-factoring the whole code

Reply:
also. I think that comment is a bit off, I will modify

Reply:
done.

Reply:
That's fine, I just wanted to touch base and see. I'm fine with deferring those changes.